pdns/dnsdist.hh

```diff
@@ -324,12 +337,10 @@ struct ClientState;
 struct IDState
 {
   IDState() : origFD(-1), sentTime(true), delayMsec(0), tempFailureTTL(boost::none) { origDest.sin4.sin_family = 0;}
-  IDState(const IDState& orig)
+  IDState(const IDState& orig): origRemote(orig.origRemote), origDest(orig.origDest)
```

Comment:
Why not also moving other member variables in initialization list ? (`origFD`, `origID`, `delayMsec` and `tempFailureTTL`)

Reply:
I had to keep the initialization of some members in the constructor body (copy constructor is missing for `std::atomic` for example), so I only changed the one that mattered, ie non-Plain Old Data types.